test/test_testing.py

```diff
@@ -58,6 +58,7 @@ def test_virtualenv(self):
 
         self.assertTrue(path_contains(sys.prefix, sys.executable))
         self.assertTrue(path_contains(sys.prefix, spam.__file__))
+        self.assertIn("VIRTUAL_ENV", os.environ)
```

Comment:
Just thinking after I've merged the PR...
Should a test like this one be added for build environments as well ? (except on Linux)